File: PythonFiles/uberDriver.py
from account import Account

class uberDriver(Account):
    name = str
    documentID = str
    email = str
    def __init__(self,name,documentID,email):
        
        Account.__init__(self,name,documentID,email)
        self.name = name
        self.documentID = documentID
        self.email = email
        def printDataDriver():
            print("----------------")
            print("Esto es uberDriver")
            print(self.name)
            print(self.email)
            print(self.documentID)
        printDataDriver()
        
             

        
        

File: PhpFiles/uberDriver.php
<?php
require_once('account.php');
class uberDriver extends account
{
    public $name;
    public $documentID;
    public $email;
    public function __construct($name,$documentID,$email)
    {
        parent::__construct($name,$documentID,$email);
        $this->$name = $name;
        $this->$documentID = $documentID;
        $this->$email = $email;
    }
    public function printDataDriver()
    {
        echo "<p>Driver Name: ".$this->name."</p>";
        echo "<p>Driver Name: ".$this->documentID."</p>";
        echo "<p>Driver Name: ".$this->email."</p>";
        
        
    }
    
}

?>